chef/cookbooks/ntp/templates/default/ntp.conf.erb

```diff
@@ -2,6 +2,8 @@
 
 driftfile <%= @driftfile %>
 
+interface ignore wildcard
+interface listen <%= @admin_interface %>
```

Comment:
No, not worth it. Was just documenting here (for myself) why it won't work on SLES11 :-)